docker/api/Dockerfile

```diff
@@ -52,7 +52,7 @@ RUN mkdir -p /opt/cnaas/templates /opt/cnaas/settings /opt/cnaas/venv \
     && chown www-data:www-data /opt/cnaas/templates /opt/cnaas/settings /opt/cnaas/venv
 
 # Copy cnaas scripts
-COPY --chown=root:www-data cnaas-setup.sh createca.sh exec-pre-app.sh nosetests.sh /opt/cnaas/
```

Comment:
I wonder why this one line has a different owner. Originally a thinko?

Reply:
Probably. There are even more small issues with the Dockfile that will only be triggered on a *nix host using a strict umask, so more PRs may come...

Reply:
The basic principle is to let www-data have write access to as absolutely little as possible, best case www-data only has write access to settings and templates and not much else. Scripts and source-code does not need to be modified at runtime so www-data should not be owner or have write access. The reason we have the www-data user and don't run everything as root is so we can separate privileges like this. If everything runs as www-data and everything is owned by www-data it's the same as everything is run by root and everything is owned by root, which we got some complaints about is not a secure solution.

Reply:
Then the best course of action is to ensure a chmod is run to give the correct privilege bits to the file, imho

Reply:
I've force-pushed changes in line with the last comment.